src/abstract/AJSingleVaultTerminal.sol

```diff
@@ -171,18 +171,22 @@ abstract contract AJSingleVaultTerminal is AJPayoutRedemptionTerminal {
                     // Perform the deposit and update the vault shares
                     // max deposit check to avoid tx being reverted
                     uint256 _maxDeposit = _vault.impl.maxDeposit(address(this));
-                    if (_maxDeposit > 0 && uint256(_targetDelta) > _maxDeposit) {
+                    if (
+                        _maxDeposit > 0 && uint256(_targetDelta) > _maxDeposit
+                    ) {
                         _vault.state.shares += _deposit(_vault, _maxDeposit);
```

Comment:
If we are depositing the `_maxDeposit` but accounting for `_targetDelta` (line 167) then the `localBalance ` becomes lower than it in reality is, which causes projects to lose funds